addons/mrp/i18n/mrp.pot

```diff
@@ -5555,6 +5555,7 @@ msgstr ""
 #. module: mrp
 #. odoo-python
 #: code:addons/mrp/models/mrp_workorder.py:0
+#: code:addons/mrp/models/mrp_workorder.py:0
```

Comment:
No need to add this line, the same one is already added just above

Reply:
I removed the line :slightly_smiling_face: